neat_usrsctp.h

```diff
@@ -1,6 +1,8 @@
 #ifndef NEAT_USRSCTP_H
 #define NEAT_USRSCTP_H
 
+#include "neat.h"
```

Comment:
This might need to be changed, if the definitions are moved from `neat.h` to `neat_internal.h`.